README.md

```diff
@@ -145,18 +145,21 @@ Read the [Locales section](#locales) for more information about supported locale
 
 ### Installation
 
-*By downloading a ZIP or a TAR.GZ...*
+#### Downloading a ZIP or tarball archive
 
-Click the github [releases tab](https://github.com/globalizejs/globalize/releases) and download the latest available Globalize package.
+Click the GitHub [releases tab](https://github.com/globalizejs/globalize/releases) and download the latest available Globalize package.
 
-*By using a package manager...*
+#### Using a package manager
 
-Use bower `bower install globalize`, or npm `npm install globalize`.
+You can use either npm and Bower:
```

Comment:
One minor grammar correction suggestion: [either ... or](https://dictionary.cambridge.org/us/grammar/british-grammar/linking-words-and-expressions/either-or):

```suggestion
You can use either npm or bower:
```

Reply:
Oops, good catch - fixed retroactively in 5b816a90eb754fd92af2ac167c0960edaebf0152.

CLA should be signed as well now.